<ui version="4.0" >
 <class>NewJobDlg</class>
 <widget class="QDialog" name="NewJobDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>-40</x>
     <y>205</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="emptyRadio" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>88</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text" >
    <string>Create Empty Job</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="profileRadio" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>15</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text" >
    <string>Create Job From Profile</string>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" native="1" name="profileWidget" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>45</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>5</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Select Profile:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
    </property>
   </widget>
   <widget class="QComboBox" name="profiles" >
    <property name="geometry" >
     <rect>
      <x>126</x>
      <y>2</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" native="1" name="emptyWidget" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>115</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>56</x>
      <y>5</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Job Name:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>13</x>
      <y>35</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Number of Tasks:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
    </property>
   </widget>
   <widget class="QLineEdit" name="jobName" >
    <property name="geometry" >
     <rect>
      <x>137</x>
      <y>1</y>
      <width>241</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="numTasks" >
    <property name="geometry" >
     <rect>
      <x>137</x>
      <y>33</y>
      <width>46</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>30</number>
    </property>
    <property name="minimum" >
     <number>1</number>
    </property>
    <property name="value" >
     <number>1</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewJobDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewJobDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
